docs/converters/vision_converter.md

```diff
@@ -5,42 +5,42 @@ SPDX-License-Identifier: MPL-2.0
 
 # Vision converter
 
-The vision converter converts the excel exports of vision to PGM data. As mentioned in [Converters](converters/converter.md), vision converter is an implementation of the tabular converter.
-The mapping of all attributes is stored in the `vision_en.yaml` and `vision_nl.yaml` files in [config](https://github.com/PowerGridModel/power-grid-model-io/tree/main/src/power_grid_model_io/config) directory.
+The Vision Excel converter converts the Excel exports from Vision to PGM data format. As mentioned in [Converters](converters/converter.md), Vision Excel converter is an implementation of the tabular converter.
+The default mapping of all attributes is stored in the `vision_en.yaml` and `vision_nl.yaml` files in [config](https://github.com/PowerGridModel/power-grid-model-io/tree/main/src/power_grid_model_io/config) directory.
+Custom mapping files are supported via passing the file directory to the constructor of the converter.
 
 ## Load rate of elements 
 
-Certain `elements` in vision, ie. appliances like transformer loads and induction motor have a result parameter of load rate.
-In vision the load rate is calculated without considering the simultaneity factor of the connected node.
-So we may observe a variation in power inflow/outflow result (ie. P,Q and S) due to different simultaneity factors. But the load rate always corresponds to `simultaneity of loads=1`.
+Certain `elements` in Vision, i.e., appliances like transformer loads and induction motor have a result parameter of load rate.
+In Vision, load rates are calculated without considering the simultaneity factors of connected nodes.
+So we may observe a variation in power inflow/outflow result (i.e., P, Q and S) due to different simultaneity factors. But the load rate always corresponds to `simultaneity of loads=1`.
 
 When we make conversion to PGM, the input data attributes of PGM for loads like `p_specified` and `q_specified` are modified as per simultaneity. The resulting loading then takes simultaneity into account. 
-**Hence, the loading of such elements may not correspond to the load rate obtained in vision**
+**Hence, the loading of such elements may not correspond to the load rate obtained in Vision**
 
 ## Transformer load modelling
 
 power-grid-model-io converts the transformer load into a individual transformer and a load for usage in power-grid-model. 
-In vision, the modelling of a transformer load seems to be different from an individual transformer and load.
-There is a minor difference in both in the reactive power consumed/generated. 
-This can correspond to a minor voltage deviation too in the results.
+In Vision, the modelling of an transformer load may seem to be different from an individual transformer and load.
+There is a minor difference in both the reactive power consumed and generated. 
+This could also correspond to a minor voltage deviation in the results.
```

Comment:
This whole section is quite vague. Was the original intention to inform users of potential cases with minor error?

Reply:
it's mainly to explain that there's deviations from the literature in Vision. we follow the literature conventions

Reply:
@petersalemink95 Perhaps you could give us some insight here?